src/cephadm/cephadm

```diff
@@ -9,6 +9,7 @@ LOG_DIR_MODE=0o770
 DATA_DIR_MODE=0o700
 CONTAINER_PREFERENCE = ['podman', 'docker']  # prefer podman to docker
 CUSTOM_PS1=r'[ceph: \u@\h \W]\$ '
+DEFAULT_TIMEOUT=None # in seconds
```

Comment:
Should we use a timeout by default or not? 

I'm a bit afraid of situations where the user has to ssh to the host to kill cephadm manually

Reply:
The appropriate timeout likely varies per service. Perhaps another PR should be created to pass a default `--timeout` from `mgr/ssh` ?